### PR TITLE
Breaks out directive sections

### PR DESCRIPTION
**Pull request by TrekSoft**
Right now we have all directives squashed together in one gooey lump. This breaks them out into delicious, bite-sized, bars.

**Note to Reviewers:** The main thing to review here is the titles I chose for the pages. There's no new content here, just breaking out the content that was in the directives page into new, separate pages.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: docs/lib/graphqlapi/fragments/js/getting-started.md [hunk 1]
```diff
@@ -176,25 +176,25 @@ When the update is complete, you can see the changes to your backend by running
...
-| @model on Object | Store objects in DynamoDB and configure CRUD resolvers. |
-| @auth on Object | Define authorization strategies for your API. | 
-| @connection on Field | Specify relationships between @model object types. |
-| @searchable on Object | Stream data of an @model object type to Amazon Elasticsearch Service. |
-| @versioned on Object | Add object versioning and conflict detection to a @model. | 
+| [@model](~/cli/graphql-transformer/model.md) on Object | Store objects in DynamoDB and configure CRUD resolvers. |
+| [@auth](~/cli/graphql-transformer/auth.md) on Object | Define authorization strategies for your API. |
+| [@connection](~/cli/graphql-transformer/connection.md) on Field | Specify relationships between @model object types. |
+| [@searchable](~/cli/graphql-transformer/searchable.md) on Object | Stream data of an @model object type to Amazon Elasticsearch Service. |
+| [@versioned](~/cli/graphql-transformer/versioned.md) on Object | Add object versioning and conflict detection to a @model. |
```
Comment (amhinson) [inline docs/lib/graphqlapi/fragments/js/getting-started.md:195]: While we're here, we should update this table with all of the possible directives.

```suggestion
| [@model](~/cli/graphql-transformer/model.md) on Object | Store objects in DynamoDB and configure CRUD resolvers. |
| [@auth](~/cli/graphql-transformer/auth.md) on Object | Define authorization strategies for your API. |
| [@connection](~/cli/graphql-transformer/connection.md) on Field | Specify relationships between @model object types. |
| [@searchable](~/cli/graphql-transformer/searchable.md) on Object | Stream data of an @model object type to Amazon Elasticsearch Service. |
| [@versioned](~/cli/graphql-transformer/versioned.md) on Object | Add object versioning and conflict detection to a @model. |
| [@key](~/cli/graphql-transformer/key.md) on Object | Index your data with keys. |
| [@function](~/cli/graphql-transformer/function.md) on Object | Connect Lambda resolvers to your API. |
| [@predictions](~/cli/graphql-transformer/predictions.md) on Object | Connect machine learning services. |
| [@http](~/cli/graphql-transformer/http.md) on Object | Configure HTTP resolvers within your API. |
```

**Review by amhinson — APPROVED** [review]

**Comment by mauerbac** [comment]
@TrekSoft - looks like there's a conflict

**Comment by TrekSoft** [comment]
Thanks - I'll address. Also need to add another file here for Rene's acceptance.

**Review by renebrandel — APPROVED** [review]

**Review by amhinson — APPROVED** [review]

**Review by dabit3 — APPROVED** [review]

**Review by renebrandel — APPROVED** [review]

**Review by wooj2 — APPROVED** [review]

**Review by jpignata — APPROVED** [review]

**Review by harrysolovay — APPROVED** [review]
LGTM

**Pull request merged**